container_pipeline/lib/registry.py

```diff
@@ -26,7 +26,7 @@ def _load_info(self):
         Loads the information about the registry to refer later.
         """
         # Get the catalog
-        catalog_url = self._registry_url + "/_catalog"
+        catalog_url = self._registry_url + "/_catalog?n=300"
```

Comment:
what happens to containers beyond number 300?
Do we need pagination URL ?

Reply:
@navidshaikh this is a temporary fix that im getting in with this. Already working on enabling pagination. I will raise another PR with pagination + revisions deletion

Reply:
what is revisions deletion?

>  Already working on enabling pagination. I will raise another PR with pagination + revisions deletion

Log an issue for this and link to the PR.

Reply:
https://github.com/CentOS/container-pipeline-service/issues/515